test/fixtures/krb5kdc-fixture/Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM ubuntu:14.04
+RUN apt update -y
+RUN apt upgrade -y
```

Comment:
np: may be  we can combine in single Docker statement ?  Doesn't matter as it is test but could save some size in docker.

Reply:
thx!!  I'll probably also make a follow on change to use `apt-get` instead of `apt`